src/App.jsx

```diff
@@ -89,6 +91,11 @@ export function App() {
     kuboIpfsAddMutation.mutate(formData);
   };
 
+  const handleKuboIpfsCatSubmit = (event) => {
+    event.preventDefault();
+    kuboIpfsCatMutation.mutate(hash);
```

Comment:
`cat` is actually query, not mutation, we read data.

Reply:
Replaced `useMutation` with `useQuery`. Using `useQuery` hook with POST method, as I understand it works okey, but it's a bit confusing, though.
I had to use `refetch()`, you were not very happy with this approach last time =/, but this is my solution, for now I will be glad for your best advice .